typings/index.test-d.ts

```diff
@@ -854,9 +854,7 @@ declare const categoryChannel: CategoryChannel;
   expectType<Promise<NewsChannel>>(categoryChannel.createChannel('name', { type: 'GUILD_NEWS' }));
   expectDeprecated(categoryChannel.createChannel('name', { type: 'GUILD_STORE' }));
   expectType<Promise<StageChannel>>(categoryChannel.createChannel('name', { type: 'GUILD_STAGE_VOICE' }));
-  expectType<Promise<TextChannel | VoiceChannel | NewsChannel | StoreChannel | StageChannel>>(
-    categoryChannel.createChannel('name', {}),
-  );
+  expectType<Promise<TextChannel>>(categoryChannel.createChannel('name', {}));
```

Comment:
Why not just this?
```suggestion
  expectType<Promise<TextChannel>>(categoryChannel.createChannel('name'));
```

Reply:
That option has always been required hence the empty object, but I presume it could be made optional

Reply:
it has always been optional iirc, better to test both actually, given the TS error

Reply:
@suneettipirneni what do you think? ^^